whatwg.org/mailing-list

```diff
@@ -27,103 +27,42 @@
-posted using the right email address and make sure you didn't send any attachments. If you still
-can't get your email sent to the list, please open a
-<a href="https://github.com/whatwg/meta/issues/new">new meta issue</a>.</p>
-
-<h3>Can I use +1?</h3>
-
-<p>Please note that content-free agreement (such as +1s) have no effect on the WHATWG list and are
-therefore discouraged. Editors of standards discussed in the WHATWG only consider the quality of
-the arguments presented, and not the volume of agreement.
-
-<p>You should therefore only post to the list if you have a substantive new point to make, for
-example if you have seen a flaw in an argument presented so far, or have a new idea to contribute,
-or have some information that has not yet been brought to the table.
-
-<h3>Should I top-post or reply inline?</h3>
-
-<p><strong>Please reply inline or make the reply self-contained, and trim extraneous quotes from
-previous emails in your replies.</strong></p>
-
-<p>Basically, please remove anything after the last line you have written, so that people don't
-have to scroll down to find out what else you wrote, and make sure that your email makes sense on
-its own, as it will probably be read out of context years later.</p>
-
-<div class=example id=example-writing-email>
- <p>That is, you should reply like this:</p>
-
- <pre>Ian wrote:
-&gt; What do you want?
-
-I want cats!
-
-&gt; When do you want it?
-
-Now!</pre>
-
- <p>You should definitely <em>not</em> reply like this (because this requires people to read your
-email backwards):</p>
-
- <pre>No
-
-Ian wrote:
-&gt; Is this a good example of how to post emails?</pre>
-
- <p>You should also <em>not</em> reply like this (because this leaves people to wonder if there is
-any text lower down that you have written):</p>
-
- <pre>This is a bad way to write email.
-
-Ian wrote:
-&gt; Is this a good way to write email?
-&gt; Lorem ipsum foo bar baz.
-&gt; Unrelated other bits that aren't replied to.
-&gt; Yet more text</pre>
-
- <p>You should also <em>not</em> reply like this (with no context at all), because the reader will
-not know what you are referring to:</p>
-
- <pre>No, I think that's a bad idea. It wouldn't be good for the readers, for instance.</pre>
-
- <p>Quote enough original text or provide an introduction yourself.</p>
-</div>
+<p>The <a href="https://lists.w3.org/Archives/Public/public-whatwg-archive/">whatwg@whatwg.org
+archives</a> maintained by the W3C are publicly accessible.</p>
+
+<p>Partials archive of whatwg@whatwg.org as well as "help" and "implementors" lists are available
+in the
+<a href="https://web.archive.org/web/20140401085615/http://lists.whatwg.org/listinfo.cgi">Wayback
+Machine</a>.</p>
+
+<p>Given a link to lists.whatwg.org, you can try finding it in the Wayback Machine:</p>
+
+<form>
+ <input name=listsurl type=url value="http://lists.whatwg.org/listinfo.cgi" size=80>
+ <input type=submit value=Go>
+</form>
+
+<script>
```

Comment:
Consider wrapping the whole thing in a `(() => {})()` (requires Edge) or `(function () { })()` (supports IE) wrapper so as not to create global variables?